spacetemplate/assets/legacy.yaml

```diff
@@ -211,6 +229,8 @@ work_item_type_groups:
   - *fundamentalID
   - *papercutsID
   bucket: portfolio
+  description: Work item type group that focuses on the planning-oriented work
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed